ui/src/app/pages/workspace/workspace-edit.spec.tsx

```diff
@@ -78,9 +78,6 @@ describe('WorkspaceEdit', () => {
     // Ensure the 'drug development' checkbox is not checked when creating.
     expect(wrapper.find('[data-test-id="researchPurpose-checkbox"]').first().prop('checked'))
       .toEqual(false);
-
-    expect(wrapper.find('[data-test-id="specific-population-no"]').first().prop('checked'))
-      .toEqual(true);
```

Comment:
can we keep this and change true to false? just to be sure that neither yes nor NO is selected by default?